### PR TITLE
parseAttributesImprovments

### PR DESCRIPTION
**Pull request by burner**
to accept html attributes common in angular2

https://angular.io/cheatsheet

replacement of https://github.com/rejectedsoftware/vibe.d/pull/1377

### CODE REVIEW

File: source/diet/parser.d [hunk 1]
```diff
@@ -285,6 +393,71 @@ unittest { // test CTFE-ability
...
+			str[0 .. e].strip(),
+			str[e + 1 .. $].strip()
+		);
+	} else {
+		return HTMLAttribute(
+			str.strip(),
+			"true"
+		);
+	}
+}*/
```
Comment (s-ludwig) [inline source/diet/parser.d]: ^ Still some left-over code

**Comment by s-ludwig** [comment]
Generally looks good. I think it could make sense to make the clamp-matching more strict (e.g. disallow "]foo"), but that's probably not that important in practice.

File: source/diet/parser.d (hunk 1 above)
Comment (burner) [inline source/diet/parser.d]: removed

**Comment by burner** [comment]
@s-ludwig maybe, but there are some weird attribute identifier out there. I rather add more tests to see if we accept real world syntax.

**Comment by burner** [comment]
thanks

**Pull request merged**